src/web/app/src/components/SearchInput/SearchInput.tsx

```diff
@@ -61,27 +62,31 @@ const SearchInput = ({ text, setText, labelFor, clickEvent }: SearchInputInterfa
 
   return (
     <div className={classes.wrapper}>
-      <TextField
-        variant="outlined"
-        size="medium"
-        fullWidth
-        label={labelFor}
-        onChange={(event) => setText(event.target.value)}
-        value={text}
-        InputProps={{
-          classes: {
-            root: classes.customInput,
-            focused: classes.customInput,
-            notchedOutline: classes.customInput,
-          },
-        }}
-        InputLabelProps={{
-          classes: {
-            root: classes.customInputText,
-            focused: classes.customInputText,
-          },
-        }}
-      />
+      {labelFor === 'Look for an Author' ? (
+        <AuthorInput text={text} setText={setText} labelFor={labelFor} />
```

Comment:
Is author input the box with the autocomplete results?

Reply:
It's the box for searching for an author, but with autocomplete results as part of that box